git/00.setup.md

```diff
@@ -42,8 +42,8 @@ combinations.
 ```bash
 git config --global alias.aa add --all
 git config --global alias.d diff --staged
-git config --global alias.count shortlog -sn
-git config --global alias.l log --pretty=format:%C(yellow)%h%Cblue%d%Creset %s %C(white) %an, %ar%Creset --graph
+git config --global alias.pf push --force-with-lease
+git config --global alias.l log --graph --decorate --oneline
```

Comment:
I guess I'm wondering if there is a better place to post/store configs that have used as a reference.  Having both, for now, might be fine.  I'll try not to be attached to all my code examples 😄